extensions/data-workspace/src/services/workspaceService.ts

```diff
@@ -257,4 +270,30 @@ export class WorkspaceService implements IWorkspaceService {
 			ProjectProviderRegistry.registerProvider(extension.exports, extension.id);
 		}
 	}
+
+	/**
+	 * Adds the specified project to list of projects to hide in projects viewlet. This list is kept track of in the workspace file
+	 * @param projectFile uri of project to remove from projects viewlet
+	 */
+	async removeProject(projectFile: vscode.Uri): Promise<void> {
+		if (vscode.workspace.workspaceFile) {
```

Comment:
Is this possible to call when we don't have a workspaceFile? It seems like we should either let it error out or at least print something in the console when this happens since I don't know of a scenario where this should be happening

Reply:
Nope it shouldn't be possible since when you create or open a project, an untitled workspace is opened if there isn't already a workspace open. I'll remove the check since it wasn't really doing anything.